build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+import com.android.build.gradle.BaseExtension
+import java.net.URL
+
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath("com.android.tools.build:gradle:${libs.versions.gradleVersion.get()}")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${libs.versions.kotlinVersion.get()}")
+    }
+}
+
+subprojects {
+    afterEvaluate {
+        if (hasProperty("dependencies")) {
+            dependencies {
+                val implementation by configurations
+                implementation(libs.androidx.appcompat)
+                implementation(libs.stdlib.jdk8)
+                implementation(libs.arcgis.maps.kotlin)
+                implementation(libs.androidx.multidex)
+            }
+        }
+
+        extensions.findByType(BaseExtension::class.java)?.let { android ->
+            android.compileOptions {
+                sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
Why do we set the source compatibility to Java 17 initially? May talk to Shubham later.

Reply:
I will take a note of this and start conversation with Shubham when he is back next week. Leave this to Java 17 for now.